docs/integrations/digitalocean/gslb.mdx

```diff
@@ -0,0 +1,315 @@
+  done
+```
+
+You should see each VM respond with similar frequency:
+
+```json
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+```
+
+To confirm your global load balancing, head over to the [Edges dashboard](https://dashboard.ngrok.com/cloud-edge/edges) once again to confirm your tunnels are online and weighted equally at 33.33%.
+
+![The ngrok dashboard showing online, weighted tunnels](img/edge-weighted-active.png)
+
+## **Step 7** (optional): Enable the Traffic Policy module for API gateway features {#enable-traffic-policy}
+
+Your demo API is already globally load-balanced, but if you want to extend your ngrok usage even further, you can enable one or more [Traffic Policy modules](/docs/http/traffic-policy/index.mdx) on your Edge to control traffic or establish standards for how your upstream services are accessed.
+
+You have two options:
+
+- Using `curl` with the ngrok API's [`/edges/https/{edge_id}/routes/{id}/policy` endpoint](/docs/api/resources/edge-route-policy-module.mdx). For this step, you will need to have exported the `ROUTE_ID` environment variable in step 4
+
+  ```bash
+  curl \
+    -X PUT "https://api.ngrok.com/edges/https/${EDGE_ID}/routes/${ROUTE_ID}/policy" \
+    -H "Authorization: Bearer ${NGROK_API_KEY}" \
+    -H "Content-Type: application/json" \
+    -H "Ngrok-Version: 2" \
+    -d '{"enabled":true,"outbound":[{"actions":[{"type":"add-headers","config":{"headers": {"is-ngrok": "1","country": "${.ngrok.geo.country_code}"}}}],"name":"Add ngrok headers"}]}'
+  ```
+
+- In the Traffic Policy module section of your [Edge](https://dashboard.ngrok.com/cloud-edge/edges) in the ngrok dashboard, where you can click the **Edit Traffic Policy** button and then the **YAML** button and add the YAML below.
```

Comment:
In the Traffic Policy module section of your [Edge](https://dashboard.ngrok.com/cloud-edge/edges) in the ngrok dashboard, you can click the **Edit Traffic Policy** button and then the **YAML** button and add the YAML below.

Reply:
removed "where you can" and just left it "click the"